makerules/modules.mk

```diff
@@ -24,12 +24,13 @@ MAKE_TARGETS := prebuild lint build unit integration check images
 # 	$(PYTHON) -m pylint $(PYLINT_OPTIONS)
 # 	@$(TOUCH) $@
 
-PREBUILD    := $(DERIVED_MODULE_DIR)/prebuild.$(TIMESTAMP)
-LINT        := $(DERIVED_MODULE_DIR)/lint.$(TIMESTAMP)    
 BUILD       := $(DERIVED_MODULE_DIR)/build.$(TIMESTAMP)   
-UNIT        := $(DERIVED_MODULE_DIR)/unit.$(TIMESTAMP)  
-INTEGRATION := $(DERIVED_MODULE_DIR)/integration.$(TIMESTAMP)
 IMAGES      := $(DERIVED_MODULE_DIR)/images.$(TIMESTAMP)
+INTEGRATION := $(DERIVED_MODULE_DIR)/integration.$(TIMESTAMP)
+LINT        := $(DERIVED_MODULE_DIR)/lint.$(TIMESTAMP)    
+PREBUILD    := $(DERIVED_MODULE_DIR)/prebuild.$(TIMESTAMP)
+SYSTEM      := $(DERIVED_MODULE_DIR)/system.$(TIMESTAMP)  
+UNIT        := $(DERIVED_MODULE_DIR)/unit.$(TIMESTAMP)  
```

Comment:
These were previously sorted in order of execution?

Reply:
Execution order is determined by dependence, but OK.

Reply:
And they were defined in order of dependence

Reply:
Correct